### ISSUE TITLE
Akinesia of the lower extremities

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5930]
**Preferred term label:** Akinesia of the lower extremities

**Synonyms**

**Definition (free text, please give PubMed ID)** Inability to initiate changes in activity or movement and to perform ordinary volitional movements rapidly and easily in the lower extremities.

**Parent term (use hpo.jax.org/app)** Akinesia HP:0002304

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
